cpp/ql/lib/semmle/code/cpp/security/Overflow.qll

```diff
@@ -5,64 +5,27 @@
 
 import cpp
 import semmle.code.cpp.controlflow.Dominance
-// `GlobalValueNumbering` is only imported to prevent IR re-evaluation.
```

Comment:
Is this no longer true?

Reply:
Oh, I experimented with using GVN but it didn't work out well. I'll re-add that line